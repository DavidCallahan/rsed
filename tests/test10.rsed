$temp = "foo"
print "temp = --$temp--$temp--$ENV1--"
