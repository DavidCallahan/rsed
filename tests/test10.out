temp = --foo--foo--1--
